spec/models/game_spec.rb

```diff
@@ -5,14 +5,12 @@
   describe "#populate_board!" do
     it 'places the correct number of pawns on the board' do
       
-      # white_user = FactoryGirl.create(:user)
-      # black_user = FactoryGirl.create(:user)
+      white_player = FactoryGirl.create(:user)
+      black_player = FactoryGirl.create(:user)
+
+      game = FactoryGirl.create(:game, white_player: white_player, black_player: black_player )
```

Comment:
1. After making the changes to this lines 8-12, we came back to this error. 

Game#populate_board! places the correct number of pawns on the board
     Failure/Error: white_player = FactoryGirl.create(:user)
     
     ActiveRecord::RecordInvalid:
       Validation failed: Game must exist
     # ./spec/models/game_spec.rb:8:in `block (3 levels) in <top (required)>'

2. I also tried a test, by adding the association to the user in factories.rb:  'association :game' , since in our game.rb model, we have has_many :users. the error:

 Game#populate_board! places the correct number of pawns on the board
     Failure/Error: white_player = FactoryGirl.create(:user)
     
     SystemStackError:
       stack level too deep

I took out the association in factories.rb after and left it at where the code at #1 is at. 

3. One thing that I noticed is that, inside rails console, I'm not able to create a game. Could this be the issue? 

  Game.create(name: 'lalal')
   (0.2ms)  BEGIN
   (0.2ms)  ROLLBACK
=> #<Game id: nil, name: "lalal", created_at: nil, updated_at: nil, number_of_moves: nil, white_player_id: nil, black_player_id: nil, game_status: nil>